modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -12,7 +12,7 @@ class ApplicationController < ::OpenidApplicationController
 
       skip_before_action :set_tags_and_extra_context, raise: false
       before_action :validate_json_format, if: -> { request.post? }
-      before_action :verify_mpi
```

Comment:
Are we replacing `veryify_mpi()`?  Just wondering if we should delete it f it's not going to be used anymore.

Reply:
verify_mpi is still used in v0